### DISCUSSION TITLE
Please restore `Keep last state` option on `Alert state if execution error or timeout` in ngAlerts

### DISCUSSION THREAD

**Discussion by vbichov** [grafana/grafana#37842] [Alerting]
Hi, 

If possible, please reconsider the removal of `Keep last state` option on `Alert state if execution error or timeout` in ngAlerts.

There are scenarios where the underlying database is not available for some period of time. 
this outage will create an Alert/OK storm.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
